client/src/app/components/stardust/Input.tsx

```diff
@@ -20,15 +20,19 @@ interface InputProps {
      * The network in context.
      */
     readonly network: string;
+    /**
+     * Default expanded state.
+     */
+    readonly isPreExpanded?: boolean;
 }
 
 /**
  * Component which will display an Input on stardust.
  */
-const Input: React.FC<InputProps> = ({ input, network }) => {
+const Input: React.FC<InputProps> = ({ input, network, isPreExpanded }) => {
     const history = useHistory();
     const { tokenInfo } = useContext(NetworkContext);
-    const [isExpanded, setIsExpanded] = useState(false);
+    const [isExpanded, setIsExpanded] = useState(isPreExpanded ?? true);
```

Comment:
the basic output doesnt look expanded by default, we should expand all the outputs to see at first glance what is the address that spent it for inputs, and where they got sent for the output part
![image](https://github.com/iotaledger/explorer/assets/3624944/f0b2a2e4-4b16-4e56-be03-89ef3fd5074a)

Reply:
it should look like this 
![image](https://github.com/iotaledger/explorer/assets/3624944/4f7b15f9-ca1a-4e3b-8744-ed4f2d873c85)